include/fiction/algorithms/simulation/sidb/quicksim.hpp

```diff
@@ -78,6 +78,12 @@ sidb_simulation_result<Lyt> quicksim(const Lyt& lyt, const quicksim_params& ps =
     st.additional_simulation_parameters.emplace_back("iteration_steps", ps.interation_steps);
     st.additional_simulation_parameters.emplace_back("alpha", ps.alpha);
     st.physical_parameters = ps.phys_params;
+
+    if (lyt.num_cells() == 0)
+    {
+        return st;
+    }
+
     st.charge_distributions.reserve(ps.interation_steps);
```

Comment:
I know you didn't name this variable `st` but now that there have been changes made, it seems to be an inconsistency to the other two ground state algorithms where this variable is called `result`, a more fitting name in my opinion. May I ask you to adjust the naming? Many thanks!